recipes/qtawesome/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.3.3" %}
+{% set sha256 = "c3c98ee4df0133ae42d202fea20253f8746266b4541c5df4269ca4131792ce0f" %}
+
+package:
+    name: qtawesome
+    version: {{ version }}
+
+source:
+    fn: qtawesome-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/q/qtawesome/QtAwesome-{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - six
+        - python
+        - qtpy
+
+test:
+    requires:
+        - pyqt
+    imports:
+        - qtawesome
+
+about:
+    home: https://github.com/spyder-ide/qtawesome
+    license: MIT
```

Comment:
Please make sure to add the license file once available.

xref: https://github.com/spyder-ide/qtawesome/issues/51